subsys/net/lib/wifi_credentials/Kconfig

```diff
@@ -58,3 +63,36 @@ config WIFI_CREDENTIALS_BACKEND_PSA_UID_OFFSET
 	  WIFI_CREDENTIALS_BACKEND_PSA_UID_OFFSET uid.
 
 endif # WIFI_CREDENTIALS_BACKEND_PSA
+
+config WIFI_CREDENTIALS_STATIC
+	bool "Static Wi-Fi network configuration"
+	default n
```

Comment:
nit: `default n` is implicit, we usually skip it.

Reply:
removed it